packages/paste-core/components/help-text/__tests__/helptext.test.tsx

```diff
@@ -10,14 +15,84 @@ describe('HelpText variant prop', () => {
   });
 
   it('should have colorTextError', () => {
-    expect(container.find('Text').prop('color')).toEqual('colorTextError');
+    expect(container.prop('color')).toEqual('colorTextError');
   });
 });
 
 describe('HelpText marginTop prop', () => {
   const container = shallow(<HelpText marginTop="space0" />);
 
   it('should have marginTop: space0', () => {
-    expect(container.find('Flex').prop('marginTop')).toEqual('space0');
+    expect(container.prop('marginTop')).toEqual('space0');
+  });
+});
+
+describe('HelpText HTML attributes', () => {
+  it('should set element data attribute for help text', (): void => {
+    const {container} = render(<HelpText data-testid="help_text">This is help text.</HelpText>);
+
+    expect(container.querySelector('[data-paste-element="HELP_TEXT"]')).toBeInTheDocument();
+    expect(screen.getByTestId('help_text').getAttribute('data-paste-element')).toEqual('HELP_TEXT');
+  });
+
+  it('should set custom element data attribute for flex wrapper and help text', (): void => {
+    const {container} = render(
+      <HelpText element="foo" data-testid="help_text">
+        This is help text.
+      </HelpText>
+    );
+
+    expect(container.querySelector('[data-paste-element="foo"]')).toBeInTheDocument();
+    expect(screen.getByTestId('help_text').getAttribute('data-paste-element')).toEqual('foo');
+  });
+});
+
+describe('Customization', () => {
+  it('should customize help text default and error variant', (): void => {
+    render(
+      <CustomizationProvider
+        baseTheme="default"
+        elements={{
```

Comment:
Now that I merged the global constant `TestTheme` we should set `theme={TestTheme` here (with a `@ts-ignore`) so that this test doesn't break on token changes.  Same for the other test below on 78

Reply:
Oh yeah, for sure

Reply:
Updated